include/rocksdb/options.h

```diff
@@ -962,6 +962,14 @@ struct DBOptions {
   // Default: 1000
   size_t keep_log_file_num;
 
+  // Recycle log files.
+  // If true, we will reuse previously written log files for new logs,
+  // overwriting the old data.  This is more efficient because the blocks
+  // are already allocated and fdatasync does not need to update the inode
+  // after each write.
+  // Default: false
```

Comment:
It makes sense to make the default 'false' for now because this is a critical piece of code and might need some time to bake. But in the future, we should make the default to be 'true' so that we can get the perf advantage for all use-cases.

BTW, how many log-files does it recycle?  Will it always alternate between two log files?